src/scss/base/_global.scss

```
@@ -0,0 +1,103 @@
+//- Home page heading
+.home-page-product__heading{
+    @include flex-center(space-between, center);
+    .category__heading{
+        @include flex-center(flex-start, center);
+        h3{
+            @include font(20px, bold);
+            text-transform: uppercase;
+            padding-left: 10px;
+        }
+    }
+    .home-filter__page{
+        @include flex-center(flex-end, center);
+        .home-filter__page-btn{
+            display: block;
+            .home-filter__page-icon{
+                color: $text-gray;
+                width: 40px;
+                height: 40px;
+                border: 1px solid $text-black;
+                text-align: center;
+                line-height: 40px;
+                margin: 3px;
+                &:hover{
+                    color: $text-white;
+                    background-color: $text-black;
+                }
+            }
+        }
+    }
+}
+
+//- Home product
+.home-product{
```

```suggestion
.home-product {
```
Sửa hết những lỗi tương tự